### PR TITLE
Removed hideChildrenAtBeginning option from datagrid

### PR DESCRIPTION
**Pull request by wachterjohannes**
__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | https://github.com/massiveart/husky/pull/589
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-master
+    * HOTFIX      #1799 [CategoryBundle]  Fixed opened category tree on startup
```
Comment (danrot) [inline CHANGELOG.md]: Wrong PR number

Comment (wachterjohannes) [inline CHANGELOG.md]: fixed

**Comment by danrot** [comment]
What is about this openToPath option? Shouldn't that also be the same on every screen?

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**